pages/linux/nl.md

```diff
@@ -13,3 +13,11 @@
 - Number only the body lines that match a basic regular expression (BRE) pattern:
 
 `nl -b p'FooBar[0-9]' {{file}}`
+
+- Number all lines including blank lines:
+
+`nl -b a {{file}}`
+
+- Read from standard output:
```

Comment:
Perhaps this should go higher up? We generally try to order the examples on a page by the order that the typical user might want to use them.

Reply:
Yes, make this the 2nd example and remove {{options}}. Btw, this is another ["useless example of cat"](http://novosial.org/shell/useless-cat/). Ideally, you would do `nl {{options}} - < {{file}}`

Reply:
I just wish to demonstrate in order to take the lines from `stdin` we need to do `nl -` instead of just `nl`.

Reply:
That is fine. Please just make this the 2nd example.